base/linalg/dense.jl

```diff
@@ -508,7 +508,7 @@ function exp!(A::StridedMatrix{T}) where T<:BlasFloat
     end
     ilo, ihi, scale = LAPACK.gebal!('B', A)    # modifies A
     nA   = norm(A, 1)
-    I    = eye(T,n)
+    Inn    = Matrix{T}(I, n)
```

Comment:
Great example why exporting single-character names is a bad idea :)

Reply:
On the other hand, in all recent work related to `I` I have found only two such collisions IIRC, indicating that in practice this issue is not so great :).